README.md

```diff
@@ -1345,6 +1345,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   
 - ### **S**
 
+  - [Sanajit Jana](https://github.com/sanajitjana)
```

Comment:
Please insert your name in line 1380. Name should be in alphabetical order.

Reply:
Sure!